docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -1,43 +1,25 @@
-Then, configure link local IP routing between the old and new subnets to allow
-old and new service deployments to communicate. This involves using {OpenStackPreviousInstaller}
-mechanism on pre-adopted cluster to configure additional link local routes
-there. This will allow EDP deployment to reach out to adopted nodes using their
-old subnet addresses.
+You must size the new subnet appropriately to accommodate the new control
+plane services. There are no specific requirements for the
+existing deployment allocation pools that are already consumed by the {OpenStackShort} environment.
 
-The new subnet should be sized appropriately to accommodate the new control
-plane services, but otherwise doesn't have any specific requirements as to the
-existing deployment allocation pools already consumed. Actually, the
-requirements as to the size of the new subnet are lower than in the second
-scenario, as the old subnet ranges are kept for the adopted nodes, which means
-they don't consume any IP addresses from the new range.
+In the following procedure, you configure `NetworkAttachmentDefinition` custom resources (CRs) to use a different subnet from what is configured in the `network_config` section of the `OpenStackDataPlaneNodeSet` CR for the same networks. The range in the `NetworkAttachmentDefinition` CR is used for control plane services, while the new range in the `OpenStackDataPlaneNodeSet` CR is used to manage IP Address Management (IPAM) for data plane nodes.
 
-In this scenario, you will configure `NetworkAttachmentDefinition` custom resources (CRs) to use a
-different subnet from what will be configured in `NetConfig` CR for the same
-networks. The former range will be used for control plane services,
-while the latter will be used to manage IPAM for data plane nodes.
+The values that are used in the following procedure are examples. Use values that are specific to your configuration.
 
-During the process, you will need to make sure that adopted node IP addresses
-don't change during the adoption process. This is achieved by listing the
-addresses in `fixedIP` fields in `OpenstackDataplaneNodeSet` per-node section.
+[IMPORTANT]
+Do not change {OpenStackShort} node IP addresses during the adoption process. List the IP addresses in the `fixedIP` fields for each node entry in the `nodes` section of the `OpenStackDataPlaneNodeSet` CR.
 
----
+.Procedure
 
-Before proceeding, configure host routes on the adopted nodes for the
-control plane subnets.
-
-To achieve this, you will need to re-run `openstack overcloud node provision` with additional `routes` entries added to `network_config`. (This change should be applied for every adopted node configuration.) For example, you may add the following to `net_config.yaml`:
-
-```yaml
+. In the `NetworkAttachmentDefinition` CR, configure host routes on the {OpenStackShort} nodes for the control plane subnets. For example:
++
+----
 network_config:
```

Comment:
This does not look like a `NetworkAttachmentDefinition`[1] at all, this looks like config for os-net-config.

[1] https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#networkattachmentdefinition-with-json-cni-config

Reply:
CC: @booxter

Reply:
Yes, as commented in https://github.com/openstack-k8s-operators/data-plane-adoption/pull/641#discussion_r1805124736, there were changes to the document that changed the desired meaning. Now that this is merged, someone from Networking team should sit down and re-read the whole section to catch such issues.